web/impact/impact/tests/test_graphql.py

```diff
@@ -0,0 +1,83 @@
+# MIT License
+# Copyright (c) 2017 MassChallenge, Inc.
+
+import json
+from django.urls import reverse
+from impact.tests.factories import ExpertFactory
+from impact.tests.api_test_case import APITestCase
+
+
+class TestGraphQL(APITestCase):
+    url = reverse('graphql')
+
+    def test_query_with_expert(self):
+        with self.login(email=self.basic_user().email):
+            user = ExpertFactory()
+            query = """
+                query {{
+                    expertProfile(id: {id}) {{
+                        user {{
+                            firstName
+                            lastName
+                            email
+                        }}
+                        phone
+                        linkedInUrl
+                        twitterHandle
+                        personalWebsiteUrl
+                        image
+                        title
+                        company
+                        primaryIndustry {{
+                            name
+                        }}
+                        additionalIndustries {{
+                            name
+                        }}
+                        bio
+                        homeProgramFamily {{
+                            name
+                        }}
+                    }}
+                }}
+            """.format(id=user.expertprofile.id)
+            response = self.client.post(self.url, data={'query': query})
+            self.maxDiff = None
+            self.assertJSONEqual(
```

Comment:
Testing every field seems excessive. Probably would be better to test that we can query an ExpertProfile and get back just the one or two fields that would confirm we were getting the right profile back.

Reply:
@johnschoeman addressed in https://github.com/masschallenge/impact-api/pull/165/commits/d65fb539a3551a1cb5a753ac6d2ffe754d034c92